docs/running from source.md

```diff
@@ -69,6 +69,20 @@ It should be dropped as "SNI" into the root folder of the project. Alternatively
 host.yaml at your SNI folder.
 
 
+## Optional: Git
+
+[Git](https://git-scm.com) is required to install some of the packages that Archipelago depends on.
+It may be possible to run Archipelago from source without it, at your own risk.
+
+It is also generally recommended to have Git installed and understand how to use it, especially if you're thinking about contributing.
+Git will allow you to keep your copy of the repository updated without having to redownload the .zip manually.
+It lets you manage branches or forks of the repository with ease, and it will be all but required to submit your changes via a pull request.
+
+You can download the latest release of Git at [The downloads page on the Git website]https://git-scm.com/downloads).
+
+Beyond that, there are also graphical interfaces for Git that make it more accessible.
+For repositories on Github (such as this one), [Github Desktop](https://desktop.github.com) is one such option.
```

Comment:
Can we (also) suggest the git integration of PyCharm? Getting people to use PyCharm is good.

Reply:
I did the thing